common/librato/librato.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
this doesn't need to be in common unless you also plan on writing an eventer sink.